kogito-kafka-quickstart-springboot/README.md

```diff
@@ -59,28 +118,66 @@ java -jar target/kogito-kafka-quickstart-sprintboot-{version}.jar
 
 To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* To examine ProcessedTravellers topic and verify upcoming messages will be processed
+
+```
+kafka_2.11-2.4.0/bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers 
+```
+
+* Send Message to Topic
+```
+kafka_2.11-2.4.0/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
 ```
-{ 
-"firstName" : "Jan", 
-"lastName" : "Kowalski", 
-"email" : "jan.kowalski@example.com", 
-"nationality" : "Polish"
-}
 
+Content (cloud event format)
+```
+{
+  "specversion": "0.3",
+  "id": "21627e26-31eb-43e7-8343-92a696fd96b1",
+  "source": "",
+  "type": "VisaApplicationsMessageDataEvent_8", 
```

Comment:
type should be `TravellersMessageDataEvent_3` (see Quarkus)

Reply:
fixed @radtriste